viewer_server/client_db_access_server.py

```diff
@@ -3,22 +3,26 @@
 #   This file is distributed under the University of Illinois Open Source
```

Comment:
This file is modified heavily and cleaned up in the authentication pull request. Might help to skip the modifications now to merge/rebase more easily with the new features.